### ISSUE TITLE
Schedule monthly hangout for collaborators 

### ISSUE THREAD

**Issue by hackygolucky** [knode/knode.github.io#10]
As the collaborators are in different cities, we really like each other, and it would be helpful to share in-person conversations we've had and not documented yet(we're human, after all), I think it would be helpful to have a monthly G+ hangout to talk shop. 

The biggest barrier to this is the number of people involved: talky.io and G+ hangouts both seem to fall over at ~5 connections. Do we have any other recommended options?

**Comment by rosskukulinski** [comment]
How important do you think it is to have video, @hackygolucky?

**Comment by hackygolucky** [comment]
Good point. Not vital. People being able to talk would be nice even if just
audio. I appreciate the opportunity to be able to move beyond something
such as IRC. The density of text in a heated discussion can get
overwhelming and reduce focus.

**Comment by daviddias** [comment]
This would be awesome, I've been following knode for a while, but it would be great to get a grasp of the whole vision :)
